package-lock.json

```diff
@@ -9,584 +9,28 @@
       "version": "0.1.11",
       "license": "Apache-2.0",
       "devDependencies": {
-        "bikeshed-to-ts": "^1.2.0",
+        "bikeshed-to-ts": "file:../bikeshed-to-ts",
```

Comment:
how does this work? is it pointing at a local checkout somewhere? I thought npm would check it out for you but is suspect.

Reply:
Oh. That was only for local testing, it shouldn't have made it into this patch. Let me make sure that's scrubbed out.

Reply:
For local testing you can use `npm link` which shouldn't affect the package-lock, I think.

Reply:
Good to know in the future! Resolved this issue.